master/tasks/main.yml

```diff
@@ -34,14 +34,6 @@
     group: _dxpb_import
     mode: 0755
 
-- name: Obtain initial Git checkout for {{ dxpb_master_repo_giturl }}
```

Comment:
This needs to create the dir, previously git was doing that, now it isn't.

Reply:
Look up one state, does that nothandle it?

Reply:
No, that is in the slave role, this is the master role.